### ISSUE TITLE
Variable Domain Modeling

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepbiologic#6]
http://onlinelibrary.wiley.com/doi/10.1002/prot.24756/full

Would be worth reimplementing, but looks like data uses proprietary dataset.

**Comment by ChinmayRout9040895625** [comment]
I agree with @rbharath —the methods used for Variable Domain Modeling in this paper are definitely worth investigating and re-implementing.

Since the original training data is proprietary, we should focus on extracting the key physicochemical features and the model architecture described. We can then train and benchmark the re-implemented model using the PNAS dataset (from Issue #7) as our gold standard for clinical-stage developability risks.
